src/Kernel.php

```diff
@@ -105,7 +108,7 @@ private function disableDebugClassLoader()
     protected function getPHPDIContainer()
     {
         if ($this->phpdiContainer === null) {
-            $builder = new \DI\ContainerBuilder();
+            $builder = new DiContainerBuilder();
```

Comment:
For clarity I'd rather not alias the class (and the namespace is really short anyway), could you restore that?

Reply:
Ok, no problem